changes/202.changed

```diff
@@ -0,0 +1 @@
+- Apply changes from breaking PR on Substra (#405(https://github.com/Substra/substra/pull/405)) ([#202](https://github.com/Substra/substrafl/pull/202))
```

Comment:
duplicate link

Reply:
it to point to the substra PR that initiate the breaking change

Reply:
my bad